geolocation/android/src/main/java/com/capacitorjs/plugins/geolocation/GeolocationPlugin.java

```diff
@@ -172,10 +172,12 @@ public void error(String message) {
     @SuppressWarnings("MissingPermission")
     private void startWatch(final PluginCall call) {
         int timeout = call.getInt("timeout", 10000);
+        int interval = call.getInt("interval", 5000);
```

Comment:
The default interval here should be the original hardcoded value (10000), so that nothing changes for those who update without having this option set.